src/java_bytecode/Makefile

```diff
@@ -3,6 +3,7 @@ SRC = bytecode_info.cpp \
       ci_lazy_methods.cpp \
       ci_lazy_methods_needed.cpp \
       expr2java.cpp \
+      generic_arguments_name_buildert.cpp \
```

Comment:
I think in general the files do not end on `t`, only the types

Reply:
Originally I had it without the `t` but Linter was complaining about it. It said something like - class names need to end with `t`.

Reply:
The linter is wrong if it requires the file to end in a `t`, but the type definitely should (i.e. the type and the file end up with different names)

Reply:
I see, will do, thanks!